roles/k8_cluster_services_rc_setup/tasks/main.yml

```diff
@@ -25,45 +25,57 @@
 
   - name: insert private registry ip in webserver controller image yml
     replace:
-      dest=/root/db-rc.yml
-      regexp='PRIVATE'
-      replace='{{ ansible_docker0.ipv4.address }}'
+      dest: /root/db-rc.yml
+      regexp: 'PRIVATE'
+      replace: '{{ ansible_docker0.ipv4.address }}'
 
   - name: insert private registry ip in db controller image yml
     replace:
-      dest=/root/webserver-rc.yml
-      regexp='PRIVATE'
-      replace='{{ ansible_docker0.ipv4.address }}'
+      dest: /root/webserver-rc.yml
+      regexp: 'PRIVATE'
+      replace: '{{ ansible_docker0.ipv4.address }}'
 
   - name: wait for kubernetes apiserver to be available
     wait_for:
       port: 8080
       timeout: 120
 
+  - name: Define kubectl path for F27AH
+    when: (ansible_distribution == "Fedora" and ansible_distribution_major_version > '26') or
+          (ansible_distribution == "CentOSDev")
+    set_fact:
+      kubectl_path: "/usr/local/bin/kubectl"
+
+  - name: Define kubectl path for F26AH, Centos/RHEL
+    when: (ansible_distribution == "RedHat") or
+          (ansible_distribution == "Fedora" and ansible_distribution_major_version < '27') or
+          (ansible_distribution == "CentOS")
+    set_fact:
+      kubectl_path: "/usr/bin/kubectl"
```

Comment:
Can we move these set_facts into a separate role and call them as a dependency of the role?  As the k8 rpms get removed from the streams, the logic will only need to be changed in a single role as opposed to several places.

Reply:
Ah dang it, I missed this piece in my latest commits....